server/controllers/finance/reports/generalLedger/index.js

```diff
@@ -66,22 +66,23 @@ function renderAccountSlip(req, res, next) {
 
   let report;
 
-  try {
-    report = new ReportManager(ACCOUNT_SLIP_TEMPLATE, req.session, options);
-  } catch (e) {
-    return next(e);
-  }
-
   const data = {};
 
-  return AccountReport.getAccountTransactions(params.account_id, GENERAL_LEDGER_SOURCE)
+  return Accounts.lookupAccount(params.account_id)
+    .then((account) => {
+      _.extend(data, { account });
+
+      options.filename = 'Extrait'.concat('_', account.number, '_', account.label);
```

Comment:
Do you think this should include a timestamp (or a date)?  That way a user can generate multiple without overriding the previous?